src/components/navigation/link.ts

```diff
@@ -182,7 +182,9 @@ export class Link {
     const forceBrand = forceBrandDomain();
 
     const domain =
-      !isInternal || userType === UserType.Guest
+      !isInternal ||
+      !userType ||
```

Comment:
you can drop `!userType` as it's not part of the array below

Reply:
If I drop it, I have a type error as userType can be undefined